core/frontend/src/Viewport.ts

```diff
@@ -1304,6 +1304,32 @@ export abstract class Viewport implements Disposable, TileUser {
     removals.push(settings.onTimePointChanged.addListener(scheduleChanged));
     removals.push(style.onScheduleScriptChanged.addListener(scriptChanged));
 
+
+    const scheduleEditingChanged = (
+      changes: Array<{ timeline: RenderSchedule.ModelTimeline, elements: Set<Id64String> }>
+    ) => {
+      for (const { timeline, elements } of changes) {
+        for (const ref of this.getTileTreeRefs()) {
+          const tree = ref.treeOwner.tileTree;
+          if (tree instanceof IModelTileTree && tree.modelId === timeline.modelId) {
+            tree.onScheduleEditingChanged({ changedElementIds: elements });
+          }
+        }
+      }
+    };
+
+    const scheduleEditingCommitted = () => {
+      for (const ref of this.getTileTreeRefs()) {
+        const tree = ref.treeOwner.tileTree;
+        if (tree instanceof IModelTileTree) {
```

Comment:
Similar advice as for `scheduleEditingChanged` - new @internal method `TileTree.onScheduleEditingCommitted`.